stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/Network.java

```diff
@@ -33,8 +34,10 @@
   private final NetworkPeerManager mNetworkPeerManager;
   private final ResponseBodyFileManager mResponseBodyFileManager;
 
-  public Network(Context context) {
-    mNetworkPeerManager = NetworkPeerManager.getOrCreateInstance(context);
+  public Network(Context context, AsyncPrettyPrinterRegistry asyncPrettyPrinterRegistry) {
+    mNetworkPeerManager = NetworkPeerManager.getOrCreateInstance(
+        context,
+        asyncPrettyPrinterRegistry);
```

Comment:
As I see this, it's clear to me that the `AsyncPrettyPrinterRegistry` should indeed be locally owned by `NetworkPeerManager`, not by `Network`.